packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -42,6 +43,35 @@ You can add any HTML, CSS, or JavaScript to the sign-in page and also customize
 
 > **Note:** Before you can customize for redirect authentication, you must customize your [Okta URL domain](/docs/guides/custom-url-domain/).
 
+## Content Security Policy (CSP) for your custom domain
+
+If you have a [custom domain](/docs/guides/custom-url-domain/main/), you need to customize the default [CSP](https://content-security-policy.com/) for your org. Okta has a default CSP that can break customizations to the sign-in page and error pages. CSP customizations only take effect on custom domains.
+
+To analyze and detect potentially malicious IP addresses that seek to bypass your CSP, use [Okta ThreatInsight](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-threatinsight).
+
+> **Note:** If you add too many trusted origins to your custom CSP, the HTTP header size can exceed the limit allowed by some server software. You may need to update the default server limits or reduce the number of trusted origins in Okta.
+
+> **Note:** If you add a trusted origin URL that redirects to a different URL, you need to include the redirect URL on the trusted origin list.
+
+### Include a nonce reference
```

Comment:
empty section

Reply:
I suppose a candidate for Maurice's new shared component (or our old global component). Just an fyi (I'd wait to see what comes of Maurice's efforts before implementing.)